src/main/java/org/spongepowered/api/util/command/dispatcher/SimpleDispatcher.java

```diff
@@ -283,9 +283,19 @@ public synchronized boolean removeMappings(Collection<?> c) {
     }
 
     @Override
-    public synchronized Optional<CommandMapping> get(String alias) {
+    public synchronized Optional<CommandMapping> resolveMapping(String alias, CommandSource source) {
         return Optional.fromNullable(this.commands.get(alias.toLowerCase()));
     }
+    
+    @Override
+    public synchronized Set<CommandMapping> getAll(String alias) {
+        Set<CommandMapping> set = new HashSet<CommandMapping>();
+        Optional<CommandMapping> mapping = Optional.fromNullable(this.commands.get(alias.toLowerCase()));
```

Comment:
No reason to wrap this in an Optional to just unwrap it again. Just check for whether it's null or not.